lib/perl5db.pl

```diff
@@ -2691,25 +2691,19 @@ sub _cmd_l_plus {
 }
 
 sub _cmd_l_calc_initial_end_and_i {
-    my ($spec, $start_match, $end_match) = @_;
-
-    # Determine end point; use end of file if not specified.
-    my $end = ( !defined $start_match ) ? $max :
-    ( $end_match ? $end_match : $start_match );
+    my ($spec, $current_line, $start_match, $end_match) = @_;
```

Comment:
`$spec` is no longer used, and if this function no longer needs `$spec` then _cmd_l_range() also no longer needs `$spec` that I can see.

Reply:
Yes, definitely doesn't. I've cleaned that up and fixed _cmd_l_subname so it gets passed the subname instead of reaching out into the caller's namespace.